Sim/SimDelphesInterface/src/DelphesSimulation.cpp

```diff
@@ -88,28 +132,28 @@ StatusCode DelphesSimulation::initialize() {
     }
   }
 
-  // Read Delphes configuration card
+  // Read Delphes configuration card (deleted by finalize())
   m_confReader = new ExRootConfReader;
```

Comment:
Why don't you use smart-pointers here? Same question applies to the other raw pointer members.